### DISCUSSION TITLE
OSP suite version 11 not getting started

### DISCUSSION THREAD

**Discussion by Gautamvj** [Open-Systems-Pharmacology/Forum#1655] [Questions & Problems]
Hello all, I installed OSP Suite version 11 on my Windows parallel desktop of macOS. However, when I am trying to run PK-Sim or MoBi it is showing me the following error
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/107406825/be50b62c-11c8-4132-b9bb-3abb60f00a0d)

How can I overcome this error? Any specific process to follow during installation?

**Comment by Yuri05** [comment]
please check this discussion: https://github.com/Open-Systems-Pharmacology/Forum/discussions/1542
